### PR TITLE
cloudformation: add timeout_in_minutes attribute

### LINKED ISSUE #36293: Add support for Timeout (--timeout-in-minutes) parameter in Cloudformation module
##### ISSUE TYPE
 
- Bug Report

##### COMPONENT NAME

Cloudformation module

##### ANSIBLE VERSION

```
ansible 2.4.2.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/ubuntu/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.12 (default, Nov 20 2017, 18:23:56) [GCC 5.4.0 20160609]

```

##### CONFIGURATION

No changes

##### OS / ENVIRONMENT

Ubuntu 16.04

##### SUMMARY

I believe this is a bug, that Ansible Cloudformation module does not support important Timeout parameter (--timeout-in-minutes key in aws-cli 'create-stack' call).

(Documentation:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-add-tags.html )

##### STEPS TO REPRODUCE

Check module documentation

##### EXPECTED RESULTS

Timeout parameter is supported

##### ACTUAL RESULTS

Timeout parameter is not supported

### PR DESCRIPTION
**Pull request by julienvey**
Added timeout_in_minutes attribute to cloudformation module
* No default value
* Only applies on stack creation

##### SUMMARY
fixes #36293

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME

cloudformation

##### ANSIBLE VERSION

```
2.6.0
```

### CODE REVIEW

**Comment by tedder** [comment]
I'm going to hold off on approval so this can be tested by a few people.

File: lib/ansible/modules/cloud/amazon/cloudformation.py
Comment (ryansb) [inline lib/ansible/modules/cloud/amazon/cloudformation.py]: I'd prefer the name `create_timeout` since this only applies on create, and I wouldn't want users to mix it up with how long the module itself waits for the stack, but is a stack-level timeout.

**Review by ryansb — COMMENTED** [review]
Looks good otherwise - minor naming request.

File: lib/ansible/modules/cloud/amazon/cloudformation.py
Comment (julienvey) [reply lib/ansible/modules/cloud/amazon/cloudformation.py]: Done

**Review by ryansb — APPROVED** [review]

**Review by s-hertel — APPROVED** [review]

**Comment by s-hertel** [comment]
Tested, works for me. Rebased since there was a conflict after #36909 was merged.

**Pull request merged**
